src/components/StyleDetail.tsx

```diff
@@ -88,6 +93,36 @@ class StyleDetail extends React.PureComponent<StyleDetailProps> {
     );
   }
 
+  renderGroupedExamples(content: ExampleWithGroups): JSX.Element {
+    return (
+      <p>
+        {content.group.map((example, index, array) => {
+          const {
+            '@_label': label,
+            '#text': text,
+          } = example;
+          return (
+            <span key={cyrb53(label)}>
+              <strong>{label}:</strong>{' '}
+              {text}{(index + 1) < array.length && '; '}
+            </span>
+          );
+        })}
+      </p>
+    );
+  }
+
+  renderExamples(content: string | ExampleWithGroups | undefined): JSX.Element | undefined {
```

Comment:
An unnecessary duplication of `renderSection` ... but not seeing a good way to overload the method like I would if this were Java, and I'd hate to over-stuff `renderSection` itself with an Examples-specific switch.

(Also: this is the part where I'm like "Am I over-stuffing this component?")